### ISSUE TITLE
I cant enter in my pokemon go acc

### ISSUE THREAD

**Issue by Juan44** [finndev/PokeBuddy#1043]
Its say "FAILED TO GET GAME DATA FROM THE SERVER (Purple button says retry) " i tried deleting the app and reinstalled and other things i cant play in my phone :'( 

Someone help me pls :'( 

Here a pic : http://www.se7ensins.com/forums/threads/failed-to-get-game-data-from-the-servers.1550232/

**Comment by AlexCtasy** [comment]
No one can help, it's a perma ban..

**Comment by Juan44** [comment]
@AlexCtasy bro but a few hours i was playing good

**Comment by Lucidrous24** [comment]
@AlexCtasy  you just said there isnt perma ban in previous posts and even if it is why they dont tell us in mail response?

**Comment by kevmuffin** [comment]
Seems to be banned, 
Niantic started to ban tons of accounts that have been using bots or fake gps etc

**Comment by Lucidrous24** [comment]
when you get banned you ge thte warning like every online game where are niantic warnings? they even troll us in mail with auto-responses XD

**Comment by Juan44** [comment]
:( i can enter in other accounts but in mine not :'(

**Comment by AlexCtasy** [comment]
@kevmuffin Sorry to tell you that but, they are just banninng bot ppl. About fake gps is not banning. And i prove it.

**Comment by Juan44** [comment]
@alexctasy so i get permanent ban :'( ??

**Comment by sacronor** [comment]
no, no one were baned by now (nothing confirmed) just wait 2-12h and all be ok
That happend to me last night now im botting again ;)
